src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -1724,8 +1724,6 @@ private int tryExecute(final Runnable runnable) {
             tailNext = tail.getNext();
             // tail is already consumed
             if (tailNext == tail) {
-                if (UPDATE_STATISTICS) spinMisses.increment();
```

Comment:
I think we still want to increment spin misses in these cases, right?

Reply:
Not sure TBH: is a misses because the we've been too fast or we've bee to slow? I suppose the latter..I don't know if we should consider both cases as "spin misses" or maybe yes...